priiloader/source/Input.cpp

```diff
@@ -72,19 +129,39 @@ s8 Input_Init( void )
 	gettimeofday(&_time_init, NULL);
 
 	STM_RegisterEventHandler(HandleSTMEvent);
+	r |= KEYBOARD_Init(HandleKBDEvent);
```

Comment:
why is the keyboard init in the STM handler? 
this code is run when reset or power is pressed on a regular wii

Reply:
Not sure what you mean, this is registering HandleKBDEvent for KEYBOARD_Init, I just put it next to the STM register since they are similar.

Reply:
my mistake, for a second it looked like it was in handleSTMevent, i prob missed the split xD